src/initializers/kafka/kafka.ts

```diff
@@ -25,8 +25,21 @@ export default class Kafka {
     logger.info(`Kafka connected ${brokers.join(',')}`);
   }
 
-  public async send(topic: string, message: string | Buffer, key?: string, partition?: number) {
-    const { key: _key, partition: _partition, offset } = await this.producer.send(topic, message, partition, key);
+  public async send(
+    topic: string,
+    message: string | Buffer,
+    key?: string,
+    partition?: number,
+    headers?: { [key: string]: string }[]
+  ) {
+    const { key: _key, partition: _partition, offset } = await this.producer.send(
+      topic,
+      message,
+      partition,
+      key,
+      null,
```

Comment:
what is this null here?

Reply:
That's the partition key. I haven't exposed that argument

Reply:
maybe add is argument with default value null?

Reply:
No need to expose it in this PR.